test/WebSites/InlineConstraintsWebSite/Startup.cs

```diff
@@ -12,6 +13,26 @@ public class Startup
         // Set up application services
         public void ConfigureServices(IServiceCollection services)
         {
+            services.ConfigureRouting(
+                routeOptions => routeOptions.ConstraintMap.Add(
+                    "IsbnDigitScheme10",
+                    typeof(IsbnDigitScheme10Constraint)));
+
+            services.ConfigureRouting(
+                routeOptions => routeOptions.ConstraintMap.Add(
+                    "IsbnDigitScheme13",
+                    typeof(IsbnDigitScheme10Constraint)));
+
+            services.ConfigureRouting(
```

Comment:
Not sure I understand the purpose of this extra configuration... can you add a comment that explains its purpose?